### ISSUE TITLE
ylim in panel.Column can only be set for one plot 

### ISSUE THREAD

**Issue by georg-mayer** [holoviz/hvplot#742]
Hello, 

I'm creating two hvplots which I want to show in the same panel-column. For each plot I set different values for xlim and ylim. When showing the hvplots, the xlims are kept different for each of the plots (as they should), but for the ylim the values for the first plot are also taken for the second plot. 

Example:

```
import numpy as np
import pandas as pd
import random
import panel as pn
import hvplot.pandas

df = pd.DataFrame(np.random.randint(0,10,size=(5,3)),columns=['a','b','c'])

ap = df.hvplot(x='a',ylim=(-5,10),xlim=(-5,10))
bp = df.hvplot(x='b',ylim=(0,12),xlim=(0,12))

pn.Column(ap,bp)
```
gives this:
![Screenshot_20220419_230434](https://user-images.githubusercontent.com/29097681/164097966-f7d7681e-1c85-4710-96fb-761f76315dcb.png)

Note that the x-axes' are different for both plots (as set above) but the y-axes' are identical, though they should be different. 

This also happens when I put the two plots in different columns in the same panel. 

This doesn't happen when I put it into two different columns and plot them individually, e.g. in a Jupyter Notebook. 

Besides all that: thanks for the great work! 

Cheers,
Georg

**Comment by jbednar** [comment]
When two plots are showing the same dimension ('c' on the _y_ axis, in this case), HoloViews assumes you want to compare the two plots and shares their y axes by default to make that easier. I can see why that's confusing in this case, and you can turn off that behavior by passing `shared_axes=False`:

```
import numpy as np
import pandas as pd
import random
import panel as pn
import hvplot.pandas

df = pd.DataFrame(np.random.randint(0,10,size=(5,3)),columns=['a','b','c'])

ap = df.hvplot(x='a',ylim=(-5,10),xlim=(-5,10), shared_axes=False)
bp = df.hvplot(x='b',ylim=(0,12),xlim=(0,12), shared_axes=False)

pn.Column(ap,bp)
```
<img width="677" alt="image" src="https://user-images.githubusercontent.com/1695496/164144270-57ebf43a-f1f7-48b2-ae20-98f0651ab418.png">

**Comment by philippjfr** [comment]
As @jbednar has pointed out this is not a bug, so I'm tempted to close the issue but maybe we need to update the docs to explain this.

**Comment by georg-mayer** [comment]
Thank you so much for your kind help and explanation! And sorry for spamming with the question.

**Comment by maximlt** [comment]
No worries! Next time you may first ask on [Discourse](https://discourse.holoviz.org/) :)

**Comment by jbednar** [comment]
Yes, I think the docs for `shared_axes` need to explain what happens when `lim`s are set (which is apparently that they act as minimum ranges that the shared axes have to cover). Similarly, the docs for `xlim`, `ylim`, etc. should indicate that when axes are shared, the provided limits are only minimums, not used as the actual limits.
